.travis/build.sh

```diff
@@ -21,7 +21,7 @@ export DOCKER_TAG=$COMMIT
 
 make docu_check
 if [ "${MAIN_BUILD}" = "TRUE" ] ; then
-  make findbugs
+  make spotbugs
```

Comment:
Do Spotbugs work on Java 11? Should we enable them there as well?

Reply:
Sadly no https://github.com/spotbugs/spotbugs/issues/756

Reply:
Having read that thread I think it might be better to just disable that particular check. Or rewrite the triggering code to not use try-with-resources. I doubt it will be fixed in spotbugs before some of us want to start developing using Jdk 11.